src/home_robot/home_robot/utils/point_cloud.py

```diff
@@ -15,30 +15,38 @@
 from tqdm import tqdm
 
 
-def get_pcd(xyz, rgb=None):
+def numpy_to_pcd(xyz : np.ndarray, rgb : np.ndarray = None) -> o3d.geometry.PointCloud:
+    """Create an open3d pointcloud from a single xyz/rgb pair"""
     xyz = xyz.reshape(-1, 3)
     if rgb is not None:
-        rgb = rgb.reshape(-1, 3) / 255
+        rgb = rgb.reshape(-1, 3)
     pcd = o3d.geometry.PointCloud()
     pcd.points = o3d.utility.Vector3dVector(xyz)
     if rgb is not None:
         pcd.colors = o3d.utility.Vector3dVector(rgb)
     return pcd
 
 
+def pcd_to_numpy(pcd : o3d.geometry.PointCloud) -> (np.ndarray, np.ndarray):
+    """Convert an open3d point cloud into xyz, rgb numpy arrays and return them."""
+    xyz = np.asarray(pcd.points)
+    rgb = np.asarray(pcd.colors)
+    return xyz, rgb
+
+
 def show_point_cloud(xyz, rgb=None, orig=None, R=None, save=None, grasps=[]):
     # http://www.open3d.org/docs/0.9.0/tutorial/Basic/working_with_numpy.html
     if rgb is not None:
         rgb = rgb.reshape(-1, 3)
-    #    if np.any(rgb > 1):
-    #        print("WARNING: rgb values too high! Normalizing...")
-    #        rgb = rgb / np.max(rgb)
+        if np.any(rgb > 1):
+            print("WARNING: rgb values too high! Normalizing...")
+            rgb = rgb / np.max(rgb)
 
-    pcd = get_pcd(xyz, rgb)
+    pcd = numpy_to_pcd(xyz, rgb)
     show_pcd(pcd, orig, R, save, grasps)
 
 
-def show_pcd(pcd, orig=None, R=None, save=None, grasps=[]):
+def show_pcd(pcd: o3d.geometry.PointCloud, orig=None, R=None, save=None, grasps=[]):
```

Comment:
Add typing for other inputs. Type hints are especially helpful in utils that are meant to be used multiple times by multiple users across the codebase.

Reply:
added